meta-mender-toradex-nxp/classes/image_type_mender_tezi.bbclass

```diff
@@ -0,0 +1,195 @@
+              "size": 1
+            }
+          }
+        }),
+        OrderedDict({
+          "name": "u-boot-env",
+          "content": {
+            "rawfile": {
+              "filename": "uboot.env",
+              "size": 1
+            }
+          }
+        }),
+        OrderedDict({
+          "name": "ubi",
+          "content": {
+            "rawfile": {
+              "dd_options": "bs=8M",
+<<<<<<< HEAD
+<<<<<<< HEAD
+              "filename": "%s.%s.gz" % (d.getVar("IMAGE_LINK_NAME"), d.getVar("FULL_IMAGE_SUFFIX"))
+=======
+              "filename": "%s.%s" % (d.getVar("IMAGE_LINK_NAME"), d.getVar("FULL_IMAGE_SUFFIX"))
+>>>>>>> mender-tezi: Remove bz2 compression.
+=======
+              "filename": "%s.%s.gz" % (d.getVar("IMAGE_LINK_NAME"), d.getVar("FULL_IMAGE_SUFFIX"))
+>>>>>>> mender-tezi: Add gz compression.
+            }
+          }
+        })
+    ]
+
+python rootfs_mender_tezi_json() {
+    json_file = "image-%s.json" % d.getVar('IMAGE_BASENAME')
+
+    import json
+    from collections import OrderedDict
+    from datetime import datetime
+
+    data = OrderedDict({ "config_format": d.getVar('TEZI_CONFIG_FORMAT'), "autoinstall": oe.types.boolean(d.getVar('TEZI_AUTO_INSTALL')) })
+
+    data["name"] = d.getVar('SUMMARY')
+    data["description"] = d.getVar('DESCRIPTION')
+    data["version"] = "Mender %s" % d.getVar('DISTRO_CODENAME')
+    data["release_date"] = datetime.strptime(d.getVar('SRCDATE'), '%Y%m%d').date().isoformat()
+    data["prepare_script"] = "prepare.sh"
+    data["wrapup_script"] = "wrapup.sh"
+    data["marketing"] = "marketing_mender_toradex.tar"
+    data["icon"] = "mender_toradex_linux.png"
+
+    product_ids = d.getVar('TORADEX_PRODUCT_IDS')
+    if product_ids is None:
+        bb.fatal("Supported Toradex product ids missing, assign TORADEX_PRODUCT_IDS with a list of product ids.")
+    data["supported_product_ids"] = product_ids.split()
+
+    if (d.getVar("FULL_IMAGE_SUFFIX") == "ubimg"):
+        data["mtddevs"] = rootfs_mender_tezi_rawnand(d)
+    else:
+        data["blockdevs"] = rootfs_mender_tezi_emmc(d)
+
+    with open(os.path.join(d.getVar('IMGDEPLOYDIR'), json_file), 'w') as outfile:
+        json.dump(data, outfile, indent=4)
+}
+
+IMAGE_CMD:mender_tezi () {
+    cp ${IMGDEPLOYDIR}/image-${IMAGE_BASENAME}*.json ${WORKDIR}/image-json/image.json
+
+    uboot_files=""
+    for file in ${UBOOT_BINARY_TEZI_EMMC} ${SPL_BINARY} uboot.env; do
+        if [ -f "${DEPLOY_DIR_IMAGE}/$file" ]; then
+            uboot_files="${DEPLOY_DIR_IMAGE}/$file $uboot_files"
+        fi
+    done
+
+    # The first transform strips all folders from the files
+    # The second adds back a subfolder
+    ${IMAGE_CMD_TAR} --transform='s/.*\///' \
+		     --transform 's,^,${IMAGE_LINK_NAME}/,' \
+		     -chf ${IMGDEPLOYDIR}/${IMAGE_NAME}.mender_tezi.tar \
+		     ${WORKDIR}/image-json/image.json ${DEPLOY_DIR_IMAGE}/mender-tezi-metadata/* \
+		     $uboot_files \
+<<<<<<< HEAD
```

Comment:
Another broken merge?

Reply:
Same as above.  Reverted, then fixed in the later commit.